src/metabase/db/data_migrations.clj

```diff
@@ -334,6 +336,64 @@
       (db/update-where! model {:collection_id nil}
         :collection_id (u/get-id new-collection)))))
 
+(defn- fix-click-through
```

Comment:
A comment referencing the original GitHub issue number would be helpful, so if we don't remember what the point of all this code is, we can easily go back and read the issue/this PR to understand it a bit better